cmd/lncli/main.go

```diff
@@ -65,24 +68,41 @@ func getClient(ctx *cli.Context) (lnrpc.LightningClient, func()) {
 
 func getClientConn(ctx *cli.Context, skipMacaroons bool) *grpc.ClientConn {
 	lndDir := cleanAndExpandPath(ctx.GlobalString("lnddir"))
+	dataDir := cleanAndExpandPath(ctx.GlobalString("datadir"))
```

Comment:
The code below only uses `dataDir` if `ctx.GlobalString("datadir")` has a non-empty string, but if it is empty/unset, then `cleanAndExpand` here sets `dataDir` to '.' rather than the expected return value from `filepath.Join(lndDir, defaultDataDirname)`. To avoid potential confusion in case a later code change tries to use `dataDir` when `ctx.GlobalString("datadir")` is empty, maybe refactor along these lines:
```
var dataDir string
if len(ctx.GlobalString("datadir")) == 0 {
    dataDir = filePath.Join(lndDir, defaultDataDirname)
} else {
    dataDir = cleanAndExpandPath(ctx.GlobalString("datadir"))
}
networkDir := filePath.Join(
    dataDir,
    defaultChainSubDirname,
```

Reply:
Good point. Updated and rebased @rodasmith